cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -92,18 +84,58 @@ type certManagerClient struct {
 	configClient        config.Client
 	proxy               Proxy
 	pollImmediateWaiter PollImmediateWaiter
+	embeddedCertManagerManifestVersion string
+	embeddedCertManagerManifestHash string
 }
 
 // Ensure certManagerClient implements the CertManagerClient interface.
 var _ CertManagerClient = &certManagerClient{}
 
+func getManifestHash() string {
+	yaml, err := manifests.Asset(embeddedCertManagerManifestPath)
+        if err != nil {
+                return ""
+        }
+       	return fmt.Sprintf("%x", sha256.Sum256(yaml))
+}
+
+// Images return the list of images required for installing the cert-manager.
```

Comment:
The comment does not match with the func implementation

Reply:
updated